core/src/main/kotlin/com/exactpro/th2/read/db/core/impl/DataBaseMonitorServiceImpl.kt

```diff
@@ -152,4 +154,4 @@ class DataBaseMonitorServiceImpl(
     }
 }
 
-private class TaskHolder(val job: Job, val updateListener: UpdateListener)
+private class TaskHolder(val job: Job)
```

Comment:
Please, roll back this change. Maybe it is unused for now but the task consists of the pulling job and listener for this job. And I want to keep those entities together.